www/source/blog/2017-06-07-Triage-and-Contributions.html.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Triage and Contribution Process
+date: 2017-06-07
+author: Ian Henry
+tags: Governance,Community
+category: Community
+classes: body-article
+---
+A super common question that comes up in lots of FOSS communities is "How do I contribute". While most projects have a file like a [CONTRIBUTING.md](https://github.com/habitat-sh/habitat/blob/master/CONTRIBUTING.md) or instructions in a README on how to join in on the open source fun, this question still gets asked quite frequently. This is an interesting problem to have for a group of people that are actively attempting to grow their community and base of contributors. The question project maintainers really need to ask themselves is quite simply: Why is this question coming up in the first place?
```

Comment:
I also think we should add contribution guidelines to our core-plans & scaffolding repos to help people understand the concepts and how to get started with submitting a new one.